roles/ceph-osd/tasks/scenarios/collocated.yml

```diff
@@ -57,4 +67,15 @@
     - "{{ devices }}"
   when:
     - not containerized_deployment
-    - item.0.partitions|length == 0
\ No newline at end of file
+    - item.0.partitions|length == 0
```

Comment:
looks like this is fixing a different bug, which means you need a separate commit for that.

Reply:
oops.  I must have accidentally done that with copy paste.  Will remove.